frontend/src/Views/LandingPage.tsx

```diff
@@ -0,0 +1,32 @@
+import Header from '../Commons/Header';
+import Footer from '../Commons/Footer';
+import Button from '../Commons/Button';
+import { StyledContent, StyledButtonGroup, StyledDivider } from './Styles/LandingPage.styles';
+
+const LandingPage = () => {
+  const consolePrint = () => {
+    console.log('Button Works!');
+  };
+  return (
+    <>
+      <Header />
+      <StyledContent>
+        <h1>About the nutrient calculator</h1>
+        <p>
+          For BC berry farmers, who maintain soil nutrient levels, Better Berries is a simple
+          nutrient calculator that lets you manage your crops&apos; on the go. Our application is
+          intuitive to use and mobile friendly, allowing you to manage your soil nutrients
+          effectively.
+        </p>
+      </StyledContent>
+      <StyledButtonGroup>
+        <Button size="lg" text="New Calculation" disabled={false} handleClick={consolePrint} />
+        <StyledDivider>or</StyledDivider>
+        <Button size="lg" text="Load Existing File" disabled={false} handleClick={consolePrint} />
+      </StyledButtonGroup>
+      <Footer />
```

Comment:
If you declare the `Header` and `Footer` in `App` you won't have to worry about adding it to every View you create. Especially since your Header/Footer don't take and props, so its the exact same component

Reply:
Thanks! I didn't know you can do that.